setup.py

```diff
@@ -13,7 +13,7 @@ def readme():
 
 
 INSTALL_REQUIRES = [
-    'compliance-checker>=2.0.0',
+    'compliance-checker==4.1.1',
```

Comment:
This works for GitHub Actions, but I don't think this is the place to put this pin. This would mean _anywhere_ I want to use the IMOS plugin, I can only use it with this specific version of the core checker. There's no need to be so restrictive - the plugin does work with all versions of the checker, it's just when you try to run in Py 3.5 that you get issues.

I think this line should stay as it is, and instead you could add a `pip install compliance-checker==4.1.1` in the [test.yml file](https://github.com/aodn/cc-plugin-imos/blob/e8791cd3635176c399e64db15988f00da421a358/.github/workflows/test.yml#L29), just before `pip install -r requirements.txt`

Or another option could be to add this in the constraints.txt file?
```
compliance-checker<=4.1.1;python_version=='3.5'
```

Reply:
I _think_ that's fixed it @mhidas. See what you think

Reply:
:+1: